juniper_official/Server-Monitoring/kube-metrics/check-deployment-status-replicas.rule

```diff
@@ -0,0 +1,62 @@
+healthbot {
+    topic kube-metrics {
+        rule check-deployment-status-replicas {
+            keys [ deployment namespace ];
+            synopsis "KPI to monitor zero unavailable replicasets";
+            description "Checks for zero unavailable replicasets";
+            sensor deployment-status {
+                description "Checks kube metrics deployment replica status";
+                server-monitoring {
+                    sensor-name /kube/deployment;
+                    frequency 60s;
+                }
+            }
+            field deployment {
+                sensor deployment-status {
+                    path deployment;
+                }
+                type string;
+                description "Deployment pod name";
+            }
+            field deployment_status {
```

Comment:
Avoid using underscore. Instead convert to hyphen (the only exceptions from what I recall happen to deal with iAgent and python fields, which cannot have hyphens).

Reply:
Done.